### PR TITLE
Update using-signalr-performance-counters-in-an-azure-web-role.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@guardrex  can't use character entity references in **bold**, it confuses the MT (machine translation).

part of #5348

### CODE REVIEW

File: aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ SignalR performance counters are used to monitor your app's performance in an Az
 
 1. Open Visual Studio 2015.
 
-2. In Visual Studio 2015, select **File &gt; New &gt; Project**.
+2. In Visual Studio 2015, select **File > Project**.
```
Comment (guardrex) [inline aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md]: Should this be ...
```
**File** > **Project**
```

File: aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md [hunk 2]
```diff
@@ -53,21 +53,21 @@ SignalR performance counters are used to monitor your app's performance in an Az
 
    [!code-xml[Main](using-signalr-performance-counters-in-an-azure-web-role/samples/sample1.xml)]
     
-8. Open the **Package Manager Console** from **Tools &gt; NuGet Package Manager**. Enter the following commands to install the latest version of SignalR and the SignalR utilities package:
+8. Open the **Package Manager Console** from **Tools > NuGet Package Manager**. Enter the following commands to install the latest version of SignalR and the SignalR utilities package:
```
Comment (guardrex) [inline aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md]:
```
**Tools** > **NuGet Package Manager**
```

File: aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md [hunk 3]
```diff
@@ -53,21 +53,21 @@ SignalR performance counters are used to monitor your app's performance in an Az
...
 
    [!code-powershell[Main](using-signalr-performance-counters-in-an-azure-web-role/samples/sample2.ps1)]
     
 9. Configure the app to install the SignalR performance counters into the role instance when it starts up or recycles. In **Solution Explorer**, right-click on the **WebRole1** project and select **Add &gt; New Folder**. Name the new folder *Startup*.
 
    ![Add Startup Folder](using-signalr-performance-counters-in-an-azure-web-role/_static/image5.png)
     
-10. Copy the *signalr.exe* file (added with the **Microsoft.AspNet.SignalR.Utils** package) from **&lt;project folder&gt;\SignalRPerfCounters\packages\Microsoft.AspNet.SignalR.Utils.&lt;version&gt;\tools** to the *Startup* folder you created in the previous step.
+10. Copy the *signalr.exe* file (added with the **Microsoft.AspNet.SignalR.Utils** package) from &lt;project folder&gt;\SignalRPerfCounters\packages\Microsoft.AspNet.SignalR.Utils.&lt;version&gt;\tools to the *Startup* folder you created in the previous step.
```
Comment (guardrex) [inline aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md]: Forward slashes? (and we're not going to sweat italicizing the path, correct? ... given this is complex enough with the "project folder" and "version" shoehorned in there.)
```
&lt;project folder&gt;/SignalRPerfCounters/packages/Microsoft.AspNet.SignalR.Utils.&lt;version&gt;/tools
```

File: aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md [hunk 4]
```diff
@@ -53,21 +53,21 @@ SignalR performance counters are used to monitor your app's performance in an Az
...
 
-11. In **Solution Explorer**, right-click the *Startup* folder and select **Add &gt; Existing Item**. In the dialog that appears, select *signalr.exe* and select **Add**.
+11. In **Solution Explorer**, right-click the *Startup* folder and select **Add Existing Item**. In the dialog that appears, select *signalr.exe* and select **Add**.
```
Comment (guardrex) [inline aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md]: Keep the > sign ... "Existing Item" is in a sub-contextual menu.
```
**Add** &gt; **Existing Item**
```

File: aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md [hunk 5]
```diff
@@ -53,21 +53,21 @@ SignalR performance counters are used to monitor your app's performance in an Az
...
 
     ![Add signalr.exe to project](using-signalr-performance-counters-in-an-azure-web-role/_static/image6.png)
     
-12. Right-click on the *Startup* folder you created. Select **Add &gt; New Item**. Select the **General** node, select **Text File**, and name the new item *SignalRPerfCounterInstall.cmd*. This command file will install the SignalR performance counters into the web role.
+12. Right-click on the *Startup* folder you created. Select **Add New Item**. Select the **General** node, select **Text File**, and name the new item *SignalRPerfCounterInstall.cmd*. This command file will install the SignalR performance counters into the web role.
```
Comment (guardrex) [inline aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md]: Same here ...
```
**Add** &gt; **New Item**
```

**Comment by Rick-Anderson** [comment]
@guardrex  closing. I like all your suggestions but I'll let you fix this one :) Probably easier to start over.

**Comment by guardrex** [comment]
@Rick-Anderson I'll make a note and square it away.

**Pull request closed without merging**